dev/start

```diff
@@ -55,25 +83,54 @@ function conjur_isready?() {
   return 1
 }
 
-function keycloak_isready?() {
-  for i in {1..20}
-  do
-   sleep=10
-   echo "keycloak starting logs:"
-   echo "$(docker-compose logs oidc-keycloak)"
-   output=$(docker-compose logs oidc-keycloak | grep "started" | wc -l)
-   if [ $output -ne 0 ]; then
-        echo "Keycloak server is up and ready"
-        return 0;
-   else
-        echo "Keycloak not ready yet sleep number $i for $sleep seconds"
-        sleep $sleep
-   fi
-  done
-  echo "Error with keycloak server start or it is too slow"
-  return 1
+function define_oidc_variables_in_conjur() {
```

Comment:
this function also loads the oidc policy. how about `configure_oidc_authenticator`?